src/fprime_gds/executables/cli.py

```diff
@@ -782,3 +782,41 @@ def get_arguments(self) -> Dict[Tuple[str, ...], Dict[str, Any]]:
 
     def handle_arguments(self, args, **kwargs):
         return args
+    
+class OpenMCTTelemetryPollerParser(ParserBase):
+    """ Parser for OpenMCT Server arguments  """
+
+    def get_arguments(self) -> Dict[Tuple[str, ...], Dict[str, Any]]:
+        """ Return arguments on whether to use OpenMCT, post to a specific URI, and publish telemetry at a specific frequency"""
```

Comment:
unrecognized words can be added to fprime-gds/.github/actions/spelling/expect.txt